mods/ra/maps/allies-06a/allies06a-AI.lua

```diff
@@ -106,7 +106,15 @@ ProduceVehicles = function()
 	end)
 end
 
+shouldProduce = false
 ProduceNaval = function()
+	if #Utils.Where(Map.ActorsInWorld, function(self) return self.Owner == player and self.Type == "syrd" end) < 1 and shouldProduce == false then
```

Comment:
Explicitly checking against false is quite horrible.  You should be able to do `if not shouldProduce and ...`